_posts/2021-11-8-datafusion-6.0.0.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: post
+title: Apache Arrow DataFusion 6.0.0 Release
+date: "2021-11-8 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+# Introduction
+
+[DataFusion](https://arrow.apache.org/datafusion/) is an embedded
+query engine which leverages the unique features of
+[Rust](https://www.rust-lang.org/) and [Apache
+Arrow](https://arrow.apache.org/) to provide a system that is high
+performance, easy to connect, easy to embed, and high quality.
+
+The Apache Arrow team is pleased to announce the DataFusion 6.0.0 release. This covers 4 months of development work
+and includes 122 commits from the following 28 distinct contributors.
+
+```
+# TODO update when we have a final 6.0 tag
+git shortlog -sn 5.0.0..87c8eaa datafusion datafusion-cli datafusion-examples
+    28  Andrew Lamb
+    25  Jiayu Liu
+     9  rdettai
+     8  QP Hou
+     5  carlos
```

Comment:
I changed my ID from carlos to Carlos the other day. So I believe I'm in lines 45 and 49, LOL 😄

Reply:
Thanks @xudong963 . I'll try and combine them when we release 6.0.0 (these stats will change slightly when we get a final `6.0.0` tag)

Reply:
@alamb could you help regen the stats here with 6.0.0 tag?

Reply:
> @alamb could you help regen the stats here with 6.0.0 tag?

Done, I also merged mine. Carlos & carlos -> xudong963  LOL 😁